src/main/java/org/reficio/p2/P2Mojo.java

```diff
@@ -219,10 +218,11 @@ private void initializeEnvironment() throws IOException {
         bundlesDestinationFolder = new File(buildDirectory, BUNDLES_DESTINATION_FOLDER);
         featuresDestinationFolder = new File(buildDirectory, FEATURES_DESTINATION_FOLDER);
         FileUtils.deleteDirectory(new File(buildDirectory, BUNDLES_TOP_FOLDER));
-        bundlesDestinationFolder.mkdirs();
-        featuresDestinationFolder.mkdirs();
+        FileUtils.forceMkdir(bundlesDestinationFolder);
+        FileUtils.forceMkdir(featuresDestinationFolder);
```

Comment:
mkdirs() returns false on failure. forceMkdir() throws an IOException, which is better